src/CSET/recipes/simple_mask.yaml

```diff
@@ -0,0 +1,36 @@
+category: Quick Look
+title: Example simple mask application
+description: Generates and applies a simple mask to a field for stratified analysis.
+
+steps:
+  - operator: read.read_cubes
+
+  - operator: filters.apply_mask
+    original_field:
+        operator: filters.filter_cubes
+        constraint:
+            operator: constraints.combine_constraints
+            variable_constraint:
+                operator: constraints.generate_var_constraint
+                varname: surface_air_temperature
+            cell_method_constraint:
+                operator: constraints.generate_cell_methods_constraint
+                cell_methods: []
+    masks:
+        operator: filters.generate_mask
+        mask_field:
+            operator: filters.filter_cubes
+            constraint:
+                operator: constraints.generate_var_constraint
+                varname: surface_altitude
+        condition: '>='
+        value: 500
+
+  - operator: collapse.collapse
+    coordinate: [grid_latitude, grid_longitude]
+    method: MEAN
+
+  - operator: write.write_cube_to_nc
+    overwrite: True
+
+  - operator: plot.plot_line_series
```

Comment:
Would this example recipe show the masking better if it was plotting a spatial plot?

Reply:
I've added a new recipe for to address your comment where we just plot a mask from the generate mask function. The three examples added were aimed at showing how to apply the masked data to other operators, and so a line plot is more effective. You wouldn't often show a spatial map of a masked area as you should be able to detect that from a standard spatial plot.